packages/angular/cli/utilities/log-file.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import { appendFileSync, mkdtempSync, realpathSync } from 'fs';
+import { tmpdir } from 'os';
+import { normalize } from 'path';
+
+let logPath: string | undefined;
+
+export function writeErrorToLogFile(error: Error): string {
```

Comment:
Would it be worth having a comment here that explains that the log file will only be generated if needed, and that the function returns the path to the temporary log file?

Reply:
Yes, I’ll do it later on!